.github/workflows/publish.yml

```diff
@@ -0,0 +1,157 @@
+name: Publish
+
+env:
+  CLOUDSMITH_REPO: "ibm-dev"
```

Comment:
The `ibm-dev` repo is a general spooling area. Once packages have been verified here they can be copied into the `edb` repo via the cloudsmith CLI and tagged `staging`.

To push packages to production we would add the `production` tag - such packages will then be synced to `{apt,yum,zypp}.enterprisedb.com` via an existing cron job.